config/filter.d/asterisk.conf

```diff
@@ -25,14 +25,24 @@ failregex = ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Registration from '[^']*'
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Host <HOST> failed MD5 authentication for '[^']*' \([^)]+\)$
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Failed to authenticate (user|device) [^@]+@<HOST>\S*$
             ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s hacking attempt detected '<HOST>'$
-            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s SecurityEvent="(FailedACL|InvalidAccountID|ChallengeResponseFailed|InvalidPassword)",EventTV="([\d-]+|%(iso8601)s)",Severity="[\w]+",Service="[\w]+",EventVersion="\d+",AccountID="(\d*|<unknown>)",SessionID=".+",LocalAddress="IPV[46]/(UDP|TCP|WS)/[\da-fA-F:.]+/\d+",RemoteAddress="IPV[46]/(UDP|TCP|WS)/<HOST>/\d+"(,Challenge="[\w/]+")?(,ReceivedChallenge="\w+")?(,Response="\w+",ExpectedResponse="\w*")?(,ReceivedHash="[\da-f]+")?(,ACLName="\w+")?$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> tried to authenticate with nonexistent user.+$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s <HOST> failed to authenticate as.+$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s Request from '[^']*' failed for '<HOST>:\d+' .+ No matching endpoint found$
+            ^(%(__prefix_line)s|\[\]\s*)%(log_prefix)s SecurityEvent="(FailedACL|InvalidAccountID|ChallengeResponseFailed|InvalidPassword)",EventTV="([\d-]+|%(iso8601)s)",Severity="[\w]+",Service="[\w]+",EventVersion="\d+",AccountID="(\d*|<unknown>)",SessionID=".+",LocalAddress="IPV[46]/(UDP|TCP|WS|WSS)/[\da-fA-F:.]+/\d+",RemoteAddress="IPV[46]/(UDP|TCP|WS|WSS)/<HOST>/\d+"(,Challenge="[\w/]+")?(,ReceivedChallenge="\w+")?(,Response="\w+",ExpectedResponse="\w*")?(,ReceivedHash="[\da-f]+")?(,ACLName="\w+")?$
+# These WARNINGS do not have a file attribute, as they're generated dynamicly
+            ^(%(__prefix_line)s|\[\]\s*WARNING%(__pid_re)s:?(?:\[C-[\da-f]*\])? )[^:]+: Friendly Scanner from <HOST>$
```

Comment:
After looking at random logs:

 ```
  == Using SIP RTP TOS bits 184
  == Using SIP RTP CoS mark 5
[2016-05-22 17:30:10] WARNING[14049][C-00001295]: Ext. 338:2 @ from-trunk: Friendly Scanner from 195.154.56.39
  == Spawn extension (ivr-18, s, 7) exited non-zero on 'SIP/10.10.48.2-0000132a'
  == Spawn extension (ivr-18, h, 1) exited non-zero on 'SIP/10.10.48.2-0000132a'
  == Using SIP VIDEO TOS bits 136
  == Using SIP VIDEO CoS mark 6
  == Using SIP RTP TOS bits 184
  == Using SIP RTP CoS mark 5
[2016-05-22 17:30:11] WARNING[14055][C-00001296]: Ext. 339:2 @ from-trunk: Friendly Scanner from 195.154.56.39
  == Spawn extension (ivr-18, s, 7) exited non-zero on 'SIP/10.10.48.2-0000132b'
  == Spawn extension (ivr-18, h, 1) exited non-zero on 'SIP/10.10.48.2-0000132b'
  == Using SIP VIDEO TOS bits 136
  == Using SIP VIDEO CoS mark 6
  == Using SIP RTP TOS bits 184
  == Using SIP RTP CoS mark 5
[2016-05-22 17:30:12] WARNING[14069][C-00001297]: Ext. 340:2 @ from-trunk: Friendly Scanner from 195.154.56.39
  == Spawn extension (ivr-18, s, 7) exited non-zero on 'SIP/10.10.48.2-0000132c'
  == Spawn extension (ivr-18, h, 1) exited non-zero on 'SIP/10.10.48.2-0000132c'
  == Using SIP VIDEO TOS bits 136
  == Using SIP VIDEO CoS mark 6
  == Using SIP RTP TOS bits 184
  == Using SIP RTP CoS mark 5
[2016-05-22 17:30:13] WARNING[14082][C-00001298]: Ext. 341:2 @ from-trunk: Friendly Scanner from 195.154.56.39
  == Spawn extension (ivr-18, s, 7) exited non-zero on 'SIP/10.10.48.2-0000132d'
  == Spawn extension (ivr-18, h, 1) exited non-zero on 'SIP/10.10.48.2-0000132d'
```

I think it should be treated as a pretty obvious hack attempt.